subprojects/maven/src/main/java/org/gradle/api/artifacts/maven/MavenDeployer.java

```diff
@@ -71,14 +71,18 @@
     void addProtocolProviderJars(Collection<File> jars);
 
     /**
-     * Returns whether to assign snapshots a unique version comprised of the timestamp and build number, or to use the
-     * same version each time. Defaults to true.
+     * This method no longer has any effect.
+     *
+     * @deprecated without replacement, Gradle no longer supports legacy Maven 2 metadata.
      */
+    @Deprecated
     boolean isUniqueVersion();
 
     /**
-     * Sets whether to assign snapshots a unique version comprised of the timestamp and build number, or to use the same
-     * version each time. Defaults to true.
+     * This method no longer has any effect.
+     *
+     * @deprecated without replacement, Gradle no longer supports legacy Maven 2 metadata.
      */
+    @Deprecated
     void setUniqueVersion(boolean uniqueVersion);
```

Comment:
I'm not sure that removing the functionality and leaving the method as a no-op meets the definition of "deprecation". I'm fine with this being a breaking change: is it worth leaving the method?

Reply:
You're right, I should probably just remove it.